src/styles/blocks/button.scss

```diff
@@ -0,0 +1,41 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  width: 100%;
+  height: 56px;
+
+  text-align: center;
+
+  border: 2px solid $colorBlue;
+  border-radius: 8px;
+
+  transition: all 0.3s;
```

Comment:
it's better not to use `all` in the transition, also u can add your transition-duration(0.3s) into variable because we use the same value throughout the entire landing page